README.md

```diff
@@ -180,10 +210,10 @@ WantedBy=multi-user.target⏎
 ### Upgrading to a new version
 To upgrade to a new version, go to the project root and run `git pull`.
 If something has changed on the models, run `source ./venv/bin/activate` to
-activate the python environment. Then run `./venv/bin/flask db upgrade` to
+activate the python environment. Then run `./flask db upgrade` to
 upgrade the database.
 
 Finally restart the redis task, which fills the redis database with meter
 readings and the webserver:
```

Comment:
Maybe a little clearer: "Finally restart the the web server and the redis task which fills the redis database with meter
readings:"